indico/modules/events/editing/service.py

```diff
@@ -123,3 +124,29 @@ def service_get_status(event):
     except requests.RequestException as exc:
         return {'status': None, 'error': unicode(ServiceRequestFailed(exc))}
     return {'status': resp.json(), 'error': None}
+
+
+def service_handle_editable(editable):
+    revision = editable.revisions[-1]
+    data = {
+        'files': EditingRevisionFileSchema().dump(revision.files, many=True),
+        'endpoints': {
+            'revisions': {
+                'replace': url_for('.api_replace_revision', editable.event,
+                                   contrib_id=editable.contribution_id, type=editable.type.name,
+                                   revision_id=revision.id, _external=True)
+            },
+            'file_upload': url_for('.api_upload', editable.event,
+                                   contrib_id=editable.contribution_id, type=editable.type.name,
+                                   _external=True)
+        }
+    }
+    try:
+        resp = requests.post(_build_url(editable.event, '/event/{}/contributions/{}/editing/{}'
+                                        .format(_get_event_identifier(editable.event),
+                                                editable.contribution_id, editable.type.name)),
+                             headers=_get_headers(editable.event), json=data)
```

Comment:
I think this would make it more readable:

```suggestion
        path = '/event/{}/contributions/{}/editing/{}'.format(
            _get_event_identifier(editable.event),
            editable.contribution_id,
            editable.type.name
        )
        resp = requests.post(_build_url(editable.event, path), headers=_get_headers(editable.event), json=data)
```

Reply:
Looks good!